app/routes/claim/view-claim.js

```diff
@@ -87,12 +88,19 @@ module.exports = function (router) {
   router.get('/claim/:claimId/download', function (req, res) {
     authorisation.isCaseworker(req)
 
-    var path = req.query.path
-    if (path) {
-      res.download(path)
-    } else {
-      throw new Error('No path to file provided')
+    var claimDocumentId = req.query['claim-document-id']
+    if (!claimDocumentId) {
```

Comment:
Is there any validation on the URL parameter - Is ClaimDocumentId an integer sequence?